lib/presentation/organisms/timer_organism.dart

```diff
@@ -24,6 +26,10 @@ class _TimerOrganismState extends State<TimerOrganism>
   @override
   void initState() {
     super.initState();
+    lockScreen = _prefs.getBool("isSilent") ?? lockScreen;
+    WakeLockController.instance.setWakeLock(lockScreen);
+    PlayerController.instance.setSilentState(!(_prefs.getBool("setSilentState") ?? true));
```

Comment:
You did the opposite, I asked you to change it to isSound not to setSilentState

Reply:
I apologize for the confusion. Are you asking to change `setSilentState` to `setIsSilent` and `isSound` to `isSilent`?

Reply:
I want everything to be “isSound”